templates/blog/blog-card.html

```diff
@@ -0,0 +1,48 @@
+{% load resolve %}
+<div class="col-4 blog-p-card--post">
+   {% resolve article.group groups as group %}
```

Comment:
Ideally, data should be transformed in the view function to avoid having to do logic in the template. So in this case, it would be good if the article object was handed to the template with all the group info it needed baked into it. Regardless of whether the template is the rather handicapped Django templates, or the more capable Jinja2, the heavy lifting would ideally be done higher up.

However, if changing the module is too cumbersome, I'm happy with this solution as it is for now.

Reply:
Yes, I have thought the same thing.
I will try to find a nice way of accomplishing this in the module.

Reply:
Have updated this in https://github.com/canonical-webteam/blog/pull/19 and added tests.